setup.cfg

```diff
@@ -24,3 +24,12 @@ filterwarnings =
     ignore:::docutils[.*]
     ignore:::jinja2[.*]
     ignore:::yaml[.*]
+
+[mypy]
+python_version = 3.8
+
+[mypy-configobj.*]
+ignore_missing_imports = True
```

Comment:
I couldn't find type annotations for configobj. The project looks dormant and I looked into switching to the Python configparser, but didn't because it's missing a lot of things configobj does and I don't know if Everett users are using those things. I figured I'd wait to see what happens. In the meantime, we'll ignore typing for configobj things for now.